tests/dags/providers/provider_api_scripts/test_phylopic.py

```diff
@@ -1,33 +1,42 @@
 import json
 import logging
-import os
 from functools import partial
+from pathlib import Path
 from unittest.mock import patch
 
 import pytest
 from common.licenses import get_license_info
 from providers.provider_api_scripts import phylopic as pp
 
 
-RESOURCES = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), "resources/phylopic"
-)
+RESOURCES = Path(__file__).parent.resolve() / "resources/phylopic"
```

Comment:
Just for my understanding, why is the `.resolve()` necessary here versus just using `.parent` by itself?

Reply:
I don't think it's necessary. I think I just copied it from a different file.